homeassistant/components/samsungtv/__init__.py

```diff
@@ -117,7 +121,9 @@ def stop_bridge(event):
     return True
 
 
-async def _async_create_bridge_with_updated_data(hass, entry):
+async def _async_create_bridge_with_updated_data(
+    hass: HomeAssistant, entry: ConfigEntry
+) -> Any:
```

Comment:
```suggestion
) -> SamsungTVBridge:
```

Reply:
Might be better with `SamsungTVLegacyBridge | SamsungTVWSBridge`

Reply:
Done